src/main/java/uk/gov/ons/ctp/integration/ratelimiter/client/RateLimiterClient.java

```diff
@@ -153,11 +157,82 @@ public RateLimitResponse checkRateLimit(
         throw limiterException;
       } else {
         // Something unexpected went wrong
-        log.warn("Limiter request failed");
+        log.warn("Limiter request for fulfilments failed");
         throw new CTPException(
             Fault.SYSTEM_ERROR,
             limiterException,
-            "POST request to limiter failed with http status: "
+            "POST request to limiter (for fulfilments) failed with http status: "
+                + httpStatus.value()
+                + "("
+                + httpStatus.name()
+                + ")");
+      }
+    }
+
+    return response;
+  }
+
+  /**
+   * Send webform limit request to the limiter.
+   *
+   * <p>If no limit has been breached then this method returns. If there is an internal error or if
+   * a limit is breached then an exception is thrown.
+   *
+   * @param domain is the domain to query against. This value is mandatory.
+   * @param ipAddress is the end users ip address. This value can be null, but if supplied then it
+   *     cannot be an empty string.
+   * @return The response from the rate limiter, or null if the rate limiter was not called.
+   * @throws CTPException if there is a processing error or if an invalid argument is supplied.
+   * @throws ResponseStatusException if the request to the limiter didn't return a 200. If a limit
+   *     has been breached then the exception status will be HttpStatus.TOO_MANY_REQUESTS and the
+   *     exception's reason field will contain the limiters json response.
+   */
+  public RateLimitResponse checkWebformRateLimit(Domain domain, String ipAddress)
+      throws CTPException, ResponseStatusException {
+
+    // Fail if caller doesn't meet interface requirements
+    verifyArgumentSupplied("domain", domain);
+    verifyArgumentNotEmpty("ipAddress", ipAddress);
+
+    log.with("ipAddress", ipAddress).info("Check webform rate limit");
+
+    // Skip check if RHUI has not been able to get the clients IP address
+    if (ipAddress == null) {
+      log.debug("Not calling rate limiter");
+      return null;
+    }
+
+    // Make it easy to access limiter parameters by adding to a hashmap
+    Map<String, String> params = new HashMap<String, String>();
+    params.put(DESC_REQUEST, "WEBFORM");
+    params.put(DESC_IP_ADDRESS, ipAddress);
+
+    // Create request
+    RateLimitRequest request = createWebformRateLimitRequest(domain, params);
+    log.with(request).debug("RateLimiterRequest for Webform");
+
+    // Send request to limiter, with detailed logging if we breached a limit
+    RateLimitResponse response;
```

Comment:
From here to line 244 is a copy of code from the previous method, so potential to factor this out into separate helper method.